chapters/synchronous.tex

```diff
@@ -1170,7 +1170,7 @@ \subsection{Discrete-time and Discretized Sub-Partitions}\label{clocked-discrete
 
 \subsection{Solver Methods}\label{solver-methods}
 
-The integration method associated with a discretized sub-partition is defined with a string.
+A sub-partition can have an integration method, directly associated \cref{associating-a-solver-to-a-partition} or interferred from other sub-partitions \cref{inferencing-of-solvermethod}.
```

Comment:
When compiled, this sentence becomes:
>A sub-partition can have an integration method, directly associated section 16.8.3 or interferred from
other sub-partitions section 16.8.4.

A typo in "inferred" and some punctuation is missing around section 16.8.x. 
```suggestion
A sub-partition can have an integration method, directly associated (\cref{associating-a-solver-to-a-partition}) or inferred from other sub-partitions (\cref{inferencing-of-solvermethod}).
```